<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Waddteacher</class>
 <widget class="QDialog" name="Waddteacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLineEdit" name="ID_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="email_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="specialization"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>146</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>female</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>male</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add_teacher_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Teacher</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="last_name_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="first_name_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>93</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>specialization</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Birth:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>189</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="day_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="month_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="year_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
